tests/test_response.py

```diff
@@ -211,6 +211,13 @@ def test_json2():
     response = Response()
     with pytest.raises(MemorValidationError, match=r"Invalid response structure. It should be a JSON object with proper fields."):
```

Comment:
This test case is not incorrect, but it is not strong for this purpose; it's better to add a test case that tries to load a partially correct JSON object.